docs/Automation.md

```diff
@@ -1,183 +1,196 @@
-    specified when the automation session begins. Each of these parameters
-    will be used unless they are overridden by a corresponding Tier 2 or Tier 1
-    parameter.
--   Tier 2 parameters can be specified via a .NET `Dictionary<string,string>`
-    parameter that is passed to the Start command. Each of these parameters will
-    be used for the lifetime of the automation session unless they are
-    overridden by a Tier 1 parameter.
--   Tier 1 parameters can be passed to some commands. Tier 1 parameters are
-    never overridden.
-
-#### Return Objects
-Each command will return a .NET object to summarize the result of the operation.
-These objects are detailed under each command:
-
-### Command Details
-#### Start Command
-The Start command initializes the AxeWindows engine for automation.
-It is required for the session to begin, and each session requires a
-corresponding call to the Stop command. A given test process is allowed only one
-active session at a time, and subsequent calls to Start will fail.
-
-##### Inputs
-
-###### .NET
-
-The **StartCommand.Execute** method accepts the following parameters:
+        // Create config to specifically target a process
+        var myConfigBuilder = Config.Builder.ForProcessId(1234);
 
-**Name** | **Type** | **Description**
----|---|---
-primaryConfig | `Dictionary<string, string>` | The set of Tier 2 parameters to apply during this session. See [Defined Constants](#defined-constants) for specific parameters.
-configFile | `string` | The full path to a file that contains JSON-serialized Tier 3 parameters. Can be `string.Empty`. Must point to a valid JSON-serialized file if non-empty.
+        // Optional: configure to create an A11yTest file
+        myConfigBuilder.WithOutputFileFormat(OutputFileFormat.A11yTest);
+
+        // Optional: configure to output the file to a specific directory (otherwise, current directory will be used)
+        myConfigBuilder.WithOutputFileFormat(OutputFileFormat.A11yTest);
+
+        // Ready to use config
+        var myConfig = myConfigBuilder.build();
+
+2. Create a `Scanner` object using the `ScannerFactory` object with the `Config`.
+
+        // Create scanner using myConfig
+        var scanner = ScannerFactory.CreateScanner(myConfig);
+
+3. Call  the `Scan` method on the `Scanner` object.
+
+        var scanResults;
+        try {
+            scanResults = scanner.Scan();
+        }
+        catch(AxeWindowsAutomationException e) {
+            // Get the message from an exception, if one is thrown.
+            var errorMessage = e.InnerException != null ? e.InnerException.Message : e.Message;
+        }
+
+4. Check the results.
 
-###### PowerShell
+        Console.WriteLine("Number of errors found in scan: " + scanResults.errorCount);
 
-The **Start-AxeWindows** cmdlet accepts the following optional
-parameters:
+
+A [complete code example](#example) can be found below.
+
+### Class Details
+
+#### Config.Builder
+
+##### ForProcessId (static)
```

Comment:
Not a blocker - I thought `static` was an incomplete parameter definition at first because of the parentheses

Reply:
Do you think maybe

`Static: ForProcessId` works better here?

Reply:
I think so, some other options include italics for the word static, or full method signatures such as `static Builder ForProcessId(int processId)`. I slightly prefer full signatures but am not too opinionated about it